machine_learning/src/machine_learning/Machine_learning.java

```diff
@@ -14,8 +18,13 @@
     /**
      * @param args the command line arguments
      */
-    public static void main(String[] args) {
-        // TODO code application logic here
+    public static void main(String[] args) throws FileNotFoundException {
+        File file = new File("glass.data.txt");
```

Comment:
Could/should probably change this to take a command line `arg` or a variable. That way we don't have to hardcode each file every time. Personal preference to remove dead comments from NB boilerplate. Otherwise this looks good and could be merged into master. This is the way we should work on projects. i.e. create a branch for the logic about to implement, dev out that logic, then PR the branch. This way we don't over write other's work/ can see and know what everyone is doing and review code. However, with these being short projects this isn't always applicable. But it's a nice thought. @kevin-ripley @BDDowns